src/extension.ts

```diff
@@ -135,6 +122,26 @@ export async function activateInternal(context: vscode.ExtensionContext) {
     });
 }
 
+export async function associateXmlSchema(context: vscode.ExtensionContext): Promise<void> {
+    const shouldAssociateSchema = process.env.APIM_AUTO_ASSOCIATE_SCHEMA?.toLowerCase() !== 'false';
```

Comment:
it's better to define such env into one place to make it easy to handle multi env vars.

Reply:
thanks, updated